src/common/redis_module/ray_redis_module.cc

```diff
@@ -1661,6 +1674,12 @@ int RedisModule_OnLoad(RedisModuleCtx *ctx,
     return REDISMODULE_ERR;
   }
 
+  if (RedisModule_CreateCommand(ctx, "ray.table_append",
```

Comment:
Do existing tests cover this codepath?  I only see it being used in `Connect` and `Disconnect`.

Reply:
I implemented extensive testing for the `Log` interface, but I was planning on opening it as a separate PR since it overhauls the `ObjectTable` to derive from the `Log` class.